code/game/objects/items/plushes.dm

```diff
@@ -676,3 +676,23 @@
 	item_state = "moffplush"
 	attack_verb = list("fluttered", "flapped")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+	var/souls = 0
+
+/obj/item/toy/plush/moth/suicide_act(mob/living/user)
+	user.visible_message("<span class='suicide'>[user] begins staring deeply into the eyes of [src]!  It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	if(do_after(user, 50, target = user))
+		souls++
+		switch(souls)
+			if(1 to 2)
+				desc = "A plushie depicting an unsettling mothperson. After eating [souls] [souls == 1 ? "soul" : "souls"] it's not looking so huggable now..."
+			if(3 to INFINITY)
+				desc = "A plushie depicting a creepy mothperson. It's eaten [souls] souls! I don't think I want to hug it any more!"
+				divine = TRUE
+				resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF | LAVA_PROOF
+		user.visible_message("<span class='suicide'>[src] rips the soul out of [user] and consumes it!</span>")
+		playsound(src, 'sound/hallucinations/wail.ogg', 50, TRUE, -1)
+		user.unequip_everything()
+		user.dust()
```

Comment:
mark the user as being hellbound, just for fluff reasons

Reply:
Seems needless, as the user leaves behind just remains and their equipment. Maybe if there was a body left behind I'd consider it, but I like it dusting

Reply:
nah, what I'm saying is that you should make it both remove their soul AND dust them

oh, and speaking of souls, you should probably make this return a shame suicide if your character doesn't have a soul

Reply:
If they're wearing a momento mori they will already be dusted when it's unequipped. I don't see what "removing the soul" would do in any context of this as the only thing that will be there, to be observed, is the pile of their equipment and the remains that aren't even part of the original carbon.

Reply:
life finds a way, arcane

someone will manage to work out how to come back to life after committing suicide, probably with slimepeople or podcloning or some feature that might get added in the future

just set "user.mind.hasSoul" to FALSE and you should be good